server/lib/validation/languages/ca.js

```diff
@@ -0,0 +1,38 @@
+const tokenizeWords = require('talisman/tokenizers/words/gersam');
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 14;
+
+const INVALIDATIONS = [{
+  fn: (sentence) => {
+    const words = tokenizeWords('ca', sentence);
+    return words.length < MIN_WORDS || words.length > MAX_WORDS;
+  },
+  error: `Number of words must be between ${MIN_WORDS} and ${MAX_WORDS} (inclusive)`,
```

Comment:
I think it would be easier if those errors would be translated to Catalan. Would make it possible to contribute without having to know English. What do you think? Other languages have done this already, most recently for example `yue`.